gate-keeper/packages/contracts/.env

```diff
@@ -12,13 +12,17 @@ PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 WORLD_ADDRESS=0x8a791620dd6260079bf849dc5567adc3f2fdc318
 PLAYER_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
+#Docker RPC
+RPC_URL="http://foundry:8545"
```

Comment:
This will break the deploy:local script for those that are not on the docker network, right?

Reply:
Would it be possible to remove this from the env and pass in with a special deploy:builder_env command. I know its not the nicest approach.